src/ui/components/Rating/index.js

```diff
@@ -68,9 +68,12 @@ export class RatingBase extends React.Component {
         id: `Rating-rating-${thisRating}`,
         key: `rating-${thisRating}`,
         ref: (ref) => { this.ratingElements[thisRating] = ref; },
+        title: (rating && !readOnly) ? i18n.sprintf(i18n.gettext(`Update your rating to %(thisRating)s out of 5.`), { thisRating }) :
+          i18n.sprintf(i18n.gettext(`Rate this add-on %(thisRating)s out of 5.`), { thisRating }),
       };
 
       if (readOnly) {
+        props.title = null;
```

Comment:
I'm not a huge fan of setting, then deleting props. I think it would be better to have an if statement that *adds* `title` to `props` if the right conditions are set.

Reply:
I've added another if statement to add, or update, the title of `props` if the user has previously rated the add-on. This still feels like it can be improved though so any suggestions are welcomed!